docs/cluster-configuration.md

```diff
@@ -252,6 +252,54 @@ spec:
       - mdevNameSelector: "GRID T4-1Q"
         resourceName: "nvidia.com/GRID_T4-1Q"
 ```
+
+## Storage Class for Scratch Space
+
+Administrators can Override the storage class to used for scratch space during transfer operations by setting the
+`scratchSpaceStorageClass` field under the HyperConverged `spec` field.
+
+The scratch space storage class is determined in the following order:
+
+value of scratchSpaceStorageClass, if that doesn't exist, use the default storage class, if there is no default storage
```

Comment:
to we have any validation?
what is going to happen if the user set a not existing storage class?

Reply:
As with anything else: the webhook will send it to the CDI API server. It its job to validate.

Reply:
ok